contracts/margined_insurance_fund/src/handle.rs

```diff
@@ -111,10 +117,20 @@ pub fn switch_vamm_status(
     // validate address
     let vamm_valid = deps.api.addr_validate(&vamm)?;
 
-    // switch vamm status here
-    vamm_switch(deps, vamm_valid, status)?;
+    // check that the vamm is actually stored in the vammlist
+    if !is_vamm(deps.storage, vamm_valid.clone()) {
+        return Err(ContractError::Std(GenericErr {
+            msg: "No vAMMs are stored".to_string(),
+        }));
+    }
 
-    Ok(Response::default())
+    // check that the status is switchable
+    // TODO
```

Comment:
This could be removed if we allow switching a vamm to open when its open, or vice versa